pizzeriaProject/drinks.ts

```diff
@@ -0,0 +1,27 @@
+import { TcoffeeIngridients, TcoffeeSize } from "./types";
+
+export class Coffee{
+    name:string;
+    readonly price!: number;
+    size: TcoffeeSize;
+    hasSugar: boolean;
+    additional: TcoffeeIngridients[];
+
+
+    constructor(name:string, size:TcoffeeSize, hasSugar: boolean, additional: TcoffeeIngridients[]){
```

Comment:
You can make `additional` as optional to not set in constructor, same for `pizza`
```suggestion
    constructor(name:string, size:TcoffeeSize, hasSugar: boolean, additional?: TcoffeeIngridients[]){
```

Reply:
i use additional array.length, when i calculate order price. if nothing comes to constructor, would it be as undefined? and then typescript does not like some value that can be undefined?

Reply:
You can use default value for constructor paramenter or `this.additional = additional ?? []`